packages/platforms/accton/x86-64/as5915-18x/onlp/builds/x86_64_accton_as5915_18x/module/src/sysi.c

```diff
@@ -150,3 +150,217 @@ int onlp_sysi_platform_manage_leds(void)
 
     return onlp_ledi_mode_set(ONLP_LED_ID_CREATE(LED_FAN), ONLP_LED_MODE_GREEN);
 }
+
+enum fan_duty {
+    FAN_DUTY_MAX = 100,    /* high speed */
+    FAN_DUTY_MID = 60,     /* middle speed */
+    FAN_DUTY_NOR = 30,     /* normal speed */
+    FAN_DUTY_LOW_1_3 = 0,  /* low speed for fan1 and fan3*/
+    FAN_DUTY_LOW_2_4 = 40  /* low speed for fan2 and fan4 */
+};
+
+enum thermal_threshold {
+    Tc = 78000,
+    Tm = 57000,
+    Tn = 50000,
+    Tb = 39000,
+    Ta = 32000,
```

Comment:
What does *b* and *a* stand for?

Reply:
It is the name defined in our thermal policy file provided by H/W team.
                                          | Te   | T0 | Ta | Tb | Tn | Tm | Tc
Tsensor(U44)(LM75-0x4B) | 0    | 6   | 32 | 39 | 50 | 57 | 78
Operating   Temperature  | -10 | 0   | 25 | 35 | 45 | 53 | 74